x/staking/types/codec.go

```diff
@@ -23,6 +23,9 @@ func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 	legacy.RegisterAminoMsg(cdc, &MsgBeginRedelegate{}, "cosmos-sdk/MsgBeginRedelegate")
 	legacy.RegisterAminoMsg(cdc, &MsgCancelUnbondingDelegation{}, "cosmos-sdk/MsgCancelUnbondingDelegation")
 	legacy.RegisterAminoMsg(cdc, &MsgUpdateParams{}, "cosmos-sdk/x/staking/MsgUpdateParams")
+	legacy.RegisterAminoMsg(cdc, &MsgTokenizeShares{}, "cosmos-sdk/MsgTokenizeShares")
```

Comment:
What is the rationale on doing between `cosmos-sdk/Msg...` or `cosmos-sdk/x/staking/Msg...`?

Reply:
MsgUpdateParams is a reused message string so it would conflict with other modules, this is why the prefix is needed from my knowledge.